extensions/ubuntu-noble.sh

```diff
@@ -0,0 +1,16 @@
+function add_host_dependencies__patch_deboostrap(){
+
+	echo "Patching Debootstrap to support Ubuntu Noble"
+	NOBLE_SYMLINK=/usr/share/debootstrap/scripts/noble
+	if [ -L ${NOBLE_SYMLINK} ] && [ -e ${NOBLE_SYMLINK} ]; then
+		:
+	else
+		if ! command -v sudo &> /dev/null; then
+			:
```

Comment:
out of curiosity, why do we have colon here?

Reply:
Accidental leftover. Removed.